Fix bugs in the Sweden focus tree

nmpolo opened the pull request:
- Add claims to all Baltic states in 'Assert our claims over the Baltic'
 - Fix war goal creation over Baltic States if they agree to be your puppet
 - Fix a few issues caused by referencing characters with accented names
 - Fix ideas applied by 'Offensive Focus' and 'Defensive Focus'
 - Fix wrong events being triggered by 'Permittenttrafiken'
 - Fix neutrality popularity being increased by 2500% instead of 25%
 - Fix heavy cruisers being defined as light

Just fyi, Paradox advises referring to characters using their "character name" rather than their actual name which I assume is due to issues with character encoding accented names. I've fixed the ones for Martin Eugen Ekström I encountered and one for a Finnish character but I wanted to point out that this issue may exist elsewhere in your codebase as I didn't look for other occurrences.

Oh and I should note that these changes are compatible with Barbarossa v1.11.13

Elfangor567 commented on common/ideas/sweden.txt:
```diff
@@ -344,7 +344,7 @@ ideas = {
 			}
 			
 			modifier = {
-				offence = 0.1
```
offence is correct, it is a separate modifier from army_attack_factor and affects air/naval attack as well. Same goes below for defence.

nmpolo replied:
I've been looking into this and it seems that whilst offence/defence should work, they don't. Not for land combat at least, I don't know about air/naval combat. I increased the offence/defence value significantly to test but it made no difference to anything visible on the template, division or combat views. So either it used to work and no longer does or it has an effect which isn't displayed anywhere.

Are you certain it works and hasn't been deprecated?

I'd suggest keeping my change but additionally adding the corresponding modifiers for air and naval combat. This would be helpful too as I had no idea those focuses are intended to buff your air force and navy as well as land forces. (Also, it could be cool to add a couple of focuses to choose between offence and defence for the air force and navy separately.)

Elfangor567 commented on events/SCA_swedish_events.txt:
```diff
@@ -317,7 +317,13 @@ country_event = {
 			}
 		}
-		SWE = { country_event = SCA_sweden.9 }
+		SWE = {
```
This should be in event sweden.9 instead as the effect of the option. This event has the effect tooltip.

nmpolo replied:
Ok, sure. I wasn't actually sure where the best place to put that code was. I will fix it when I can.

Elfangor567 commented on history/units/SWE_1936_naval_mtg.txt:
```diff
@@ -36,7 +36,7 @@
 			ship = { name = "HSwMS Drottning Victoria" definition = heavy_cruiser equipment = { ship_hull_cruiser_coastal_defense_ship = { amount = 1 owner = SWE version_name = "Sverige Class" } } }				
 			
-			ship = { name = "HSwMS Gotland" definition = light_cruiser equipment = { ship_hull_cruiser_2 = { amount = 1 owner = SWE version_name = "Gotland Class" } } }				
+			ship = { name = "HSwMS Gotland" definition = heavy_cruiser equipment = { ship_hull_cruiser_2 = { amount = 1 owner = SWE version_name = "Gotland Class" } } }				
```
My God, thank you. I spent more time than I am willing to admit trying to figure out what was causing those bugs to show up in the debugger.

nmpolo replied:
Glad my OCD regarding them being listed as light cruisers in the intel ledger helped haha!

Elfangor567 commented on common/national_focus/Sweden.txt:
```diff
@@ -540,13 +548,13 @@ focus_tree = {
 			}
 			hidden_effect = {
```
Replace this section with the following, it should be more accurate:
```hidden_effect = {
				if = {
					limit = { has_country_leader = { id = 13002 ruling_only = yes character = SWE_martin_ekstrom } }
					every_country = {
						limit = {
							OR = {
								original_tag = LIT
								original_tag = LAT
								original_tag = EST
							}
							has_war = no
							is_subject = no
						}
						country_event = SCA_sweden.7
					}
					every_country = {
						limit = {
							OR = {
								original_tag = LIT
								original_tag = LAT
								original_tag = EST
							}
							OR = {
								has_war = yes
								is_subject = yes
							}
						}
						SWE = {
							create_wargoal = {
								type = annex_everything
								target = PREV
							}
						}
					}
				}
				else = {
					every_country = {
						limit = {
							OR = {
								original_tag = LIT
								original_tag = LAT
								original_tag = EST
							}
						}
						SWE = {
							create_wargoal = {
								type = annex_everything
								target = PREV
							}
						}
					}
				}
			}

nmpolo replied:
Roger that.

nmpolo commented:
The IC cost reduction from 'Expand Landsverk' wasn't working so I've pushed a commit for that as well. I assume this was broken due to changes when the tank designer was added.

Just wanted to mentioned this specifically as it's possible a similar change will be needed for the latest compatibility patch with the new plane designer.

Also, I wanted to ask if it was intentional that SPG variants aren't affected by this idea? As I never played before v1.11 so I don't know whether they used to be affected or not.

nmpolo commented:
I noticed the election event for the holiday cabinet was  firing even if you get democracy support above 50% when the election occurs so I've added an alternative event for that.

Also, regarding event `neutral_election.6`, I noticed only the ruling party trigger had been changed from Paradox's `election.6` event so I changed all references of democratic to neutrality in the event. Also, I see this code uses a modifier called `can_lose_democracy_support`, however, I don't think there is a neutrality version of this so I suggest that modifier is removed for this event and to make the AI always take (or heavily weighted towards) the third option to prevent strange outcomes.

nmpolo commented:
I just realised `can_lose_democracy_support` is a scripted trigger so it would be easy enough to make an equivalent for neutrality for all the election events.

nmpolo commented:
Added a `can_lose_neutrality_support` scripted modifier to replace the `can_lose_democracy_support` mentioned  in my previous 2 comments so that's all sorted.

nmpolo commented:
Turns out the neutrality election events weren't firing due to the on_action. I've fixed this but had a question about it: is it intended that this effect all neutral countries and not just those your mod is aimed at? I'm wondering because these events will cause the Communist coalition event to fire for Yugoslavia in their 1938 election. I'm unsure what effect, if any, it will have on other countries. I can only see this being a positive thing though as it may result in Yugoslavia flipping ideology as they historically did near the end of the war. That is, if they don't already do so? I've seen a Communist Bulgaria appear during the war multiple times but I don't think I've seen Yugoslavia flip or have a civil war yet. Do you know what normally happens with Yugoslavia and what are your thoughts about having this effect all neutral countries @Elfangor567 ?

Edit: Looks like this will cause a Communist coalition in Greece in their 1936 election too. However, since BftB, Greece is explicitly excluded from all the democratic election events, in the same way the USA is prior to 1949.1.1, so I think adding the same clause to the neutrality events is a good solution for this.

Edit2: Copied over the vanilla democratic restrictions for GRE, TUR and SPR.
